client/state/data-layer/wpcom/me/devices/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { keyBy } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import {
+	USER_DEVICES_REQUEST
+} from 'state/action-types';
+import { userDevicesAdd } from 'state/user-devices/actions';
+import { errorNotice } from 'state/notices/actions';
+
+export const requestUserDevices = function( { dispatch } ) {
+	dispatch( http( {
+		apiVersion: '1.1',
+		method: 'GET',
+		path: '/notifications/devices',
+	} ) );
+};
```

Comment:
probably fine either way, but mixing styles like this is awkward… (although perfectly legitimate)

```js
export const request = ( { dispatch } ) => …

export function request( { dispatch } ) { …

// also since we're simply evaluating an expression…no curlies!
export const request = ( { dispatch } ) => dispatch( … )
```